<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Параметры модели</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QSpinBox" name="op1_delta_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="client_delta_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_26">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обработки запросов компьютером 3, мин</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="op1_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_22">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="op2_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_20">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="op3_delta_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="op4_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="comp1_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обслуживания оператора 2, мин</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обработки запросов компьютером 1, мин</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обслуживания оператора 1, мин</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Метод моделирования</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обслуживания оператора 3, мин</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="op4_delta_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_25">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обработки запросов компьютером 2, мин</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_23">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="n_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="comp2_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время прихода криентов, мин</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="op3_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>45</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_24">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Время обслуживания оператора 4, мин</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="op2_delta_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Событийный</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Количество обработанных заявок</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_21">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="client_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="comp3_spb">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Результаты</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Количество обработанных заявок</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Вероятность отказа</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Количество пропущенных заявок</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_27">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Среднее время ожидания в очереди 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="rejected_probability_line_edit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="processed_count_line_edit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rejected_count_line_edit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_28">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Среднее время ожидания в очереди 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="avg_waiting_queue_1_line_edit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="avg_waiting_queue_2_line_edit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="modeling_btn">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Моделировать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
